src/visibility.js

```diff
@@ -132,9 +142,14 @@ class Visibility extends BasePlugin {
       this._stopDrag();
     }
     this.dispatchEvent(EventType.FLOATING_PLAYER_STATE_CHANGED, {active: false});
+    const playerSizeAfterFloating = this._state.shell.playerSize;
+    if (this._playerSizeBeforeFloating !== playerSizeAfterFloating) {
+      this._store.dispatch(actions.updatePlayerClientRect(this._floatingContainer.getBoundingClientRect()));
```

Comment:
did you test the fix in different browsers ? i remember at some point the behavior of getBoundingClientRect used to be inconsistent across browsers

Reply:
tested on Safari and Chrome. either way this is how we update the `playerClientRect` in other places (getting the bounding) so if there was an issue, i guess we'd see it before.